sjb/config/test_cases/test_branch_s2i.yml

```diff
@@ -0,0 +1,16 @@
+---
+parent: 'common/test_cases/s2i.yml'
+extensions:
+  sync_repos:
+    - name: "source-to-image"
+  actions:
+    - type: "script"
+      title: "test s2i"
+      repository: "source-to-image"
+      timeout: 7200
+      script: |-
```

Comment:
You might want these to be separate stages ... once we do more with metrics it may be nice to track them independently